openpype/hosts/max/plugins/publish/extract_thumbnail.py

```diff
@@ -23,22 +21,26 @@ def process(self, instance):
         self.log.debug(
             f"Create temp directory {tmp_staging} for thumbnail"
         )
-        fps = int(instance.data["fps"])
+        ext = instance.data.get("imageFormat")
         frame = int(instance.data["frameStart"])
         instance.context.data["cleanupFullPaths"].append(tmp_staging)
-        filename = "{name}_thumbnail..png".format(**instance.data)
-        filepath = os.path.join(tmp_staging, filename)
-        filepath = filepath.replace("\\", "/")
-        thumbnail = self.get_filename(instance.name, frame)
+        filepath = os.path.join(tmp_staging, instance.name)
+
+        self.log.debug("Writing Thumbnail to '{}'".format(filepath))
 
-        self.log.debug(
-            "Writing Thumbnail to"
-            " '%s' to '%s'" % (filename, tmp_staging))
         review_camera = instance.data["review_camera"]
-        with viewport_camera(review_camera):
-            preview_arg = self.set_preview_arg(
-                instance, filepath, fps, frame)
-            rt.execute(preview_arg)
+        viewport_options = instance.data.get("viewport_options", {})
+        files = render_preview_animation(
+            filepath,
+            ext,
+            review_camera,
+            frame,
+            frame,
```

Comment:
This makes it easier to see why `frame` is passed twice.
```suggestion
            start_frame=frame,
            end_frame=frame,
```